ehcache-impl/src/main/java/org/ehcache/impl/persistence/DefaultLocalPersistenceService.java

```diff
@@ -115,6 +140,15 @@ private void internalStart() {
   @Override
   public synchronized void stop() {
     if (started) {
+      try {
+        if (cleanFile.createNewFile()) {
+          LOGGER.debug("clean file is created.");
+        } else {
+          LOGGER.warn("clean file already exists.");
```

Comment:
We should probably replace this with a more informative message. Since we're logging this at warn level what should a user do with the information? What's gone/going wrong, and what should they do to fix it?